test/test-cases/functional/saic/test_vm_to_vm_commn_acl_inbound.py

```diff
@@ -0,0 +1,139 @@
+import json
+from pathlib import Path
+from pprint import pprint
+import time
+import pytest
+import sys
+sys.path.append("../utils")
+import snappi_utils as su
+
+current_file_dir = Path(__file__).parent
+
+"""
+This covers following scenario :
+
+Test vnet to vnet communication with ACL on inbound direction:
+1. Configure DPU to deny and allow traffic
```

Comment:
Please use the term DUT instead of DPU. In the future, we may test smartswitches (which contain DPUs)